src/sentry/static/sentry/app/styles/global.tsx

```diff
@@ -62,6 +62,9 @@ const styles = (theme: Theme, isDark: boolean) => css`
   /* Override css in LESS files here as we want to manually control dark mode for now */
   ${isDark
     ? css`
+        .box {
+          background: ${theme.background};
+        }
```

Comment:
Does this impact the main app anywhere?

Reply:
i didn't comprehensively check but since `theme.background` is white in light mode, i think the worst case scenario is that dark mode black-on-white turns into black-on-gray, but that's already wrong in dark mode. i feel like we should be able to address those on a case-by-case basis as they arise.